algolia/search/index_objects.go

```diff
@@ -344,6 +344,27 @@ func (i *Index) browserForObjects(params searchParams, opts ...interface{}) func
 // prevent issues, one may want to pass the opt.Safe(true) option. Note that
 // passing this option will make the method blocking.
 func (i *Index) ReplaceAllObjects(objects interface{}, opts ...interface{}) (g *wait.Group, err error) {
+
+	safe := iopt.ExtractSafe(opts...).Get()
+
+	indexExists, err := i.Exists()
+	if err != nil {
+		err = fmt.Errorf("cannot check if the index exists: %v", err)
+		return
+	}
+
+	if !indexExists {
+		resSaveObjects, e := i.SaveObjects(objects, opts...)
+		if e != nil {
+			err = fmt.Errorf("cannot save objects to the index: %v", err)
+			return
+		}
+		if safe {
+			resSaveObjects.Wait()
```

Comment:
I think we have to check for the returned error of `Wait`.

Reply:
Done ✅